syntheticsclientv2/integration_test.go

```diff
@@ -191,366 +194,579 @@ func TestLiveGetDevicesV2(t *testing.T) {
+		return 0, err
+	}
 
 	// Make the request with your check settings and print result
-	res, _, err := c.GetHttpCheckV2(1528)
+	res, reqDetail, err := c.CreateHttpCheckV2(&inputHttpCheckV2Data)
 	if err != nil {
-		fmt.Println(err)
-	} else {
-		JsonPrint(res)
+		return 0, err
 	}
+	fmt.Printf("\nReq was: \n%v\n", reqDetail)
+	JsonPrint(res)
 
+	return res.Test.ID, nil
+}
+
+func UpdateHttpCheckV2(checkId int, test string, c *Client) error {
+
+	err := json.Unmarshal([]byte(test), &inputHttpCheckV2Data)
 	if err != nil {
-		t.Fatal(err)
+		return err
 	}
 
+	// Make the request with your check settings and print result
+	res, reqDetail, err := c.UpdateHttpCheckV2(checkId, &inputHttpCheckV2Data)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("\nReq was: \n%v\n", reqDetail)
+	JsonPrint(res)
+
+	return nil
 }
 
-func TestLiveUpdateHttpCheckV2(t *testing.T) {
+func GetHttpCheckV2(checkId int, c *Client) error {
 
-	err := json.Unmarshal([]byte(updateHttpCheckV2Body), &inputHttpCheckV2Data)
+	// Make the request with your check settings and print result
+	res, _, err := c.GetHttpCheckV2(checkId)
 	if err != nil {
-		t.Fatal(err)
+		return err
 	}
 
-	//Update your client with the token
-	c := NewClient(token, realm)
+	JsonPrint(res)
 
-	fmt.Println(inputHttpCheckV2Data)
+	return nil
+}
 
+func DeleteHttpCheckV2(checkId int, c *Client) error {
 	// Make the request with your check settings and print result
-	res, reqDetail, err := c.UpdateHttpCheckV2(3420, &inputHttpCheckV2Data)
+	res, err := c.DeleteHttpCheckV2(checkId)
 	if err != nil {
-		fmt.Println(err)
-	} else {
-		fmt.Println(reqDetail)
-		JsonPrint(res)
+		return err
 	}
 
+	JsonPrint(res)
+
+	return nil
+
+}
+
+func TestLiveCreateHttpCheckV2(t *testing.T) {
+	//Create your client with the token
+	c := NewClient(token, realm)
+
+	checkId, err := CreateHttpCheckV2(createHttpCheckV2Body, c)
 	if err != nil {
 		t.Fatal(err)
 	}
+	log.Printf("Http Check ID: %d", checkId)
 
 }
 
-func TestLiveDeleteHttpCheckV2(t *testing.T) {
+func TestLiveUpdateHttpCheckV2(t *testing.T) {
 
 	//Create your client with the token
 	c := NewClient(token, realm)
 
-	// Make the request with your check settings and print result
-	res, err := c.DeleteHttpCheckV2(2099)
+	err := UpdateHttpCheckV2(1111, updateHttpCheckV2Body, c)
 	if err != nil {
-		fmt.Println(err)
-	} else {
-		JsonPrint(res)
+		t.Fatal(err)
 	}
 
+}
+
+func TestLiveGetHttpCheckV2(t *testing.T) {
+
+	//Create your client with the token
+	c := NewClient(token, realm)
+
+	err := GetHttpCheckV2(1111, c)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 }
 
-func TestLiveCreateBrowserCheckV2(t *testing.T) {
+func TestLiveDeleteHttpCheckV2(t *testing.T) {
 
-	err := json.Unmarshal([]byte(createBrowserCheckV2Body), &inputBrowserCheckV2Data)
+	//Create your client with the token
+	c := NewClient(token, realm)
+
+	err := DeleteHttpCheckV2(1111, c)
 	if err != nil {
 		t.Fatal(err)
 	}
 
+}
+
+func TestLiveBrowserCheckCreateUpdateAndDeleteV2(t *testing.T) {
```

Comment:
End to end test of Browser Check:
1. Create
2. Update
3. Get
4. Update with original payload
5. Get
6. Delete